09_Day_Higher_order_functions/09_day_higher_order_functions.md

```diff
@@ -355,7 +356,7 @@ const scores = [
   { name: 'John', score: 100 },
 ]
 
-const scoresGreaterEight = scores.filter((score) => score.score > 80)
+const scoresGreaterEighty = scores.filter((score) => score.score > 80)
 console.log(scoresGreaterEight)
```

Comment:
If const's name is `scoresGreaterEighty`, then the line 360 must be `console.log(scoresGreaterEighty);`

Reply:
Yeah, it was changed before. You can check here: https://github.com/Asabeneh/30-Days-Of-JavaScript/pull/113/commits/7b2b7f2e1ca426a986ca58a6834b6eab9f1fe369